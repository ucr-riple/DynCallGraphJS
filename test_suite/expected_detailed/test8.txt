{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test8.js:5:1:5:4)': [
        'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test8.js:1:1:3:2)'
    ]
}