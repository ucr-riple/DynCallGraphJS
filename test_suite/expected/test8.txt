{
  '(test8.js:5:1:5:4)': [ '(test8.js:1:1:3:2)' ]
}